<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTools::Internal::ProjectSettingsWidget</class>
 <widget class="QWidget" name="ClangTools::Internal::ProjectSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="globalCustomComboBox">
       <item>
        <property name="text">
         <string>Use Global Settings</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use Customized Settings</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreGlobal">
       <property name="text">
        <string>Restore Global Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gotoGlobalSettingsLabel">
       <property name="text">
        <string>&lt;a href=&quot;target&quot;&gt;Open Global Settings&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="gotoAnalyzerModeLabel">
       <property name="text">
        <string>&lt;a href=&quot;target&quot;&gt;Go to Analyzer&lt;/a&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ClangTools::Internal::RunSettingsWidget" name="runSettingsWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Suppressed diagnostics</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="diagnosticsView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="removeSelectedButton">
            <property name="text">
             <string>Remove Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeAllButton">
            <property name="text">
             <string>Remove All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClangTools::Internal::RunSettingsWidget</class>
   <extends>QWidget</extends>
   <header>clangtools/runsettingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
